src/reducers/url-state.js

```diff
@@ -124,10 +129,41 @@ function selectedThread(state: ThreadIndex = 0, action: Action) {
   }
 }
 
-function callTreeSearchString(state: string = '', action: Action) {
+function calltreeSearchString(
+  state: SearchStringState = { current: '', stack: [] },
```

Comment:
The `COMMIT_CALL_TREE_SEARCH_STRING` action will change both properties, and move the current value of `current` into the stack; that's why I used one reducer for both values.